src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot[robot[key]]) {
+      return null;
+    }
+    invertedRobot[robot[key]] = key;
```

Comment:
When assigning the key as the value and the value as the key, it's important to ensure that the value is actually a string that can be used as a property name. If the value is not a string (e.g., a number, an object, etc.), it should be converted to a string to maintain the integrity of the object structure.